Licenses/mit.txt

```diff
@@ -0,0 +1,20 @@
+MIT License
+Copyright (c) <year> <copyright holders>
```

Comment:
@klogg do we need to put the above?

Reply:
This deserves a dedicated commit, IMO

Reply:
Please also update Licenses/README with the new license file added

Reply:
Add copyright of EPAM. For existing files retain existing copyrights